docs/data-sources/resources.md

```diff
@@ -30,44 +30,44 @@ data "twingate_resources" "foo" {
 
 ### Read-Only
 
-- `id` (String) The ID of this resource.
-- `resources` (Block List) List of Resources (see [below for nested schema](#nestedblock--resources))
+- `id` (String) The ID of the data source result
```

Comment:
Why did this change?

Reply:
@alexmensch proposed to change it like this for all Terraform IDs that we need to set manually for datasources
https://github.com/Twingate/terraform-provider-twingate/pull/328#discussion_r1235470296